Mathlib/Analysis/Normed/Group/SemiNormedGrp.lean

```diff
@@ -362,7 +358,12 @@ theorem coe_comp {M N K : SemiNormedGrp₁} (f : M ⟶ N) (g : N ⟶ K) :
 instance : Inhabited SemiNormedGrp₁ :=
   ⟨of PUnit⟩
 
-instance ofUnique (V : Type u) [SeminormedAddCommGroup V] [i : Unique V] :
+/-- This instance allows type class synthesis to "see through" `SemiNormedGrp₁.of` to
+the underlying type when attempting synthesis on `Unique (SemiNormedGrp₁.of _)`.
+
+It is scoped because its discrimination tree keys are `Unique _` allowing
+Lean to attempt it in *any* search for `Unique α`. -/
```

Comment:
What makes this one different to the others?

Reply:
Better at hiding than the others during my cursory search.